src/sp/it/pl/audio/tagging/Metadata.kt

```diff
@@ -619,21 +620,18 @@ class Metadata: Item, Serializable {
         null
     }
 
-    private fun readArt(): Artwork? {
-        val af = if (isFileBased()) getFile().readAudioFile().orNull() else null
-        return af?.tag?.firstArtwork
-    }
+    private fun readArt(): Artwork? = getFile()?.let { it.readAudioFile().orNull() }?.tag?.firstArtwork
 
     /** @return the cover image file on a file system or null if this item is not file based */
     private fun readCoverOfDir(): Cover? {
```

Comment:
```suggestion
    private fun findDirCover(): Cover? {
```
seems like a more proper name, since it does not simply read but search.

Reply:
Because the return type is Cover (whether it is loaded Image or not is +- impl detail), Id argue the search is only a part of the logic performed, but I dont mind changing it. Seems reasonable.

Reply:
There is also method called getReadCoverOfTag(), which is a typo. So, I think all three methods should have th esame prefix. if not read, find ?

Reply:
The problem with find is that the past version is found, I don't like `getFoundCoverOfTag` (what is this even lol?)
I guess just stay with read.

Reply:
getFoundCoverOfTag would become readCoverOfTag or readCoverFromTag so `find` is not wrong to use

Reply:
I much prefer `from` over `of`

Reply:
Renamed to 
- readCoverFromTag
- readCoverFromDir
- readArtworkFromTag